<launch>
  <!-- Launch AruCo Detect Qr code stuff -->
  <include file="$(find windblade_scanner)/launch/2_robot_setup/world_frame/asus_xtion_pro/asus_xtion_pro_aruco_detect_qr.launch"/>
  
  <!-- Negative red -->
  <param name="x_tf_trans" value="0.0"/>
  <!-- Negative Green-->
  <param name="y_tf_trans" value="0.0"/>
  <!-- Negative Blue -->
  <param name="z_tf_trans" value="0.0"/>

  <!-- In radians -->
  <param name="x_tf_rot" value="3.14"/>
  <param name="y_tf_rot" value="0.0"/>
  <param name="z_tf_rot" value="0.0"/>

  <!-- Launches TF publisher for world frame. -->
  <node pkg="windblade_scanner" type="world_frame_publisher.py" name="world_frame_publisher" output="screen" >

  </node>

</launch>